packages/go/cypher/analyzer/analyzer.go

```diff
@@ -72,29 +90,35 @@ func (s *ComplexityMeasure) onFunctionInvocation(node *model.FunctionInvocation)
 		// Calling for a relationship's type is highly likely to be inefficient and should add weight
 		s.Weight += Weight2
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onQuantifier(node *model.Quantifier) {
+func (s *ComplexityMeasure) onQuantifier(stack *model.WalkStack, node *model.Quantifier) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.

Reply:
Patched up